### PR TITLE
Move to internal nginx-base image for better security defaults

### PR DESCRIPTION
**Pull request by wgreenberg**

### CODE REVIEW

**Comment by vbrown608** [comment]
Thanks for updating this! We're not using travis for builds here - just to run the linter and confirm that the build is working. It would be great to keep it so we can continue getting that feedback on PRs.

**Comment by wgreenberg** [comment]
oh gotcha! I'll leave it in then

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM nginx:alpine
+FROM conex.eff.org/techops/nginx-base:latest
```
Comment (sydneyli) [inline Dockerfile]: Like we mentioned, let's add a comment here about how this is for internal use only.

Comment (wgreenberg) [reply Dockerfile]: added!

**Pull request merged**